docs/finality-provider-operation.md

```diff
@@ -887,8 +888,20 @@ This will withdraw **ALL** accumulated rewards to the address you set in the
 `set-withdraw-addr` command if you set one. If no withdrawal address was set,
 the rewards will be withdrawn to your finality provider address.
 
+
 Congratulations! You have successfully set up and operated a finality provider.
 
+### 6.7. Query Rewards
+
+You can query the reward gauges of a given stakeholder address.
+
+```shell
+fpd reward-gauges <address>
```

Comment:
Seems the `chain-id` flag is required? Better to add it here

Reply:
I tested it without that flag and worked alright:

```
❯ fpd reward-gauges bbn128fsysa3c9wqlsc8l5y23jlsg8s708ph3pqahg --node https://rpc-dapp.devnet.babylonlabs.io/
reward_gauges:
  finality_provider:
    coins:
    - amount: "963863176"
      denom: ubbn
    withdrawn_coins: []
```
But can add it as optional anyways

Reply:
Actually, when using the `chain-id` flag I get the error `unknown flag: --chain-id`

Reply:
Ah I see. Thanks for the confirmation! Maybe we should leave `--node` in the example given that it needs a connection to an RPC node